src/entropic/sinkhorn.jl

```diff
@@ -178,17 +245,23 @@ returned as three-dimensional array where `γ[:, :, i]` is the optimal transport
 
 See also: [`sinkhorn2`](@ref)
 """
-function sinkhorn(μ, ν, C, ε; kwargs...)
-    # compute Gibbs kernel
-    K = @. exp(-C / ε)
+function sinkhorn(μ, ν, C, ε, alg::Sinkhorn; kwargs...)
+    # build solver
+    solver = build_solver(alg, μ, ν, C, ε; kwargs...)
```

Comment:
Wouldn't be better to have the same order of arguments for `sinkhorn` and `build_solver`?

Reply:
Probably, again I assume it was since initially I used a different order in `sinkhorn`. I'll update it.

Reply:
Fixed.